sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessorCheckpoint.cs

```diff
@@ -39,10 +47,23 @@ public class EventProcessorCheckpoint
         ///
         public string PartitionId { get; set; }
 
+        /// <summary>
+        /// The date and time the checkpoint was last modified.
+        /// </summary>
+        ///
+        public DateTimeOffset? LastModified {  get; set; }
+
         /// <summary>
         ///   The starting position within the partition's event stream that this checkpoint is associated with.
         /// </summary>
         ///
         public EventPosition StartingPosition { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventProcessorCheckpoint"/> class.
+        /// </summary>
+        public EventProcessorCheckpoint()
```

Comment:
I don't think we need an explicit constructor, do we?

Reply:
Yeah this was leftover from my previous prototype.